admin/views/tabs/network/features.php

```diff
@@ -16,46 +16,48 @@
-<div class="yoast-measure">
-	<?php
-	echo sprintf(
-		/* translators: %s expands to Yoast SEO */
-		esc_html__( 'This tab allows you to selectively disable %s features for all sites in the network. By default all features are enabled, which allows site admins to choose for themselves if they want to toggle a feature on or off for their site. When you disable a feature here, site admins will not be able to use that feature at all.', 'wordpress-seo' ),
-		'Yoast SEO'
-	);
+<div class="tab-block">
+	<h2><?php esc_html_e( 'Features', 'wordpress-seo' ); ?></h2>
+	<div class="yoast-measure">
+		<?php
+		echo sprintf(
+			/* translators: %s expands to Yoast SEO */
+			esc_html__( 'This tab allows you to selectively disable %s features for all sites in the network. By default all features are enabled, which allows site admins to choose for themselves if they want to toggle a feature on or off for their site. When you disable a feature here, site admins will not be able to use that feature at all.', 'wordpress-seo' ),
+			'Yoast SEO'
+		);
 
-	foreach ( $feature_toggles as $feature ) {
-		$help_text = esc_html( $feature->label );
-		if ( ! empty( $feature->extra ) ) {
-			$help_text .= ' ' . $feature->extra;
-		}
-		if ( ! empty( $feature->read_more_label ) ) {
-			$help_text .= ' ';
-			$help_text .= sprintf(
-				'<a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s</a>',
-				esc_url( WPSEO_Shortlinker::get( $feature->read_more_url ) ),
-				esc_html( $feature->read_more_label )
-			);
-		}
+		foreach ( $feature_toggles as $feature ) {
+			$help_text = esc_html( $feature->label );
+			if ( ! empty( $feature->extra ) ) {
+				$help_text .= ' ' . $feature->extra;
+			}
+			if ( ! empty( $feature->read_more_label ) ) {
+				$help_text .= ' ';
+				$help_text .= sprintf(
+					'<a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s</a>',
+					esc_url( WPSEO_Shortlinker::get( $feature->read_more_url ) ),
+					esc_html( $feature->read_more_label )
+				);
+			}
 
-		$feature_help = new WPSEO_Admin_Help_Button(
-			$feature->read_more_url,
-			/* translators: %s expands to a feature's name */
-			sprintf( esc_html__( 'Help on: %s', 'wordpress-seo' ), esc_html( $feature->name ) )
-		);
+			$feature_help = new WPSEO_Admin_Help_Button(
+				$feature->read_more_url,
+				/* translators: %s expands to a feature's name */
+				sprintf( esc_html__( 'Help on: %s', 'wordpress-seo' ), esc_html( $feature->name ) )
+			);
 
-		$yform->toggle_switch(
-			WPSEO_Option::ALLOW_KEY_PREFIX . $feature->setting,
-			[
-				'on'  => __( 'Allow Control', 'wordpress-seo' ),
-				'off' => __( 'Disable', 'wordpress-seo' ),
-			],
-			'<strong>' . $feature->name . '</strong>',
-			$feature_help
-		);
-	}
-	?>
+			$yform->toggle_switch(
+				WPSEO_Option::ALLOW_KEY_PREFIX . $feature->setting,
+				[
+					'on'  => __( 'Allow Control', 'wordpress-seo' ),
+					'off' => __( 'Disable', 'wordpress-seo' ),
+				],
+				'<strong>' . $feature->name . '</strong>',
```

Comment:
If we escape `$feature->name` a few lines above within `$feature_help`, then I guess we should escape it here as well.